luthier/config/src/tables.lua

```diff
@@ -168,15 +192,26 @@ M.physical = {
 
 M.logical = {
     wikipedia = {
+        type = "abridged",
         metrics =  {"count", "added", "delta", "deleted"},
         dimensions = wikipedia_dimensions,
-        granularity = {"all", "hour", "day"},
-        physicalTables = {"wikiticker"}
+        granularities = {
+            --"all",
+            "hour",
+            "day"
+        },
+        physicalTables = {"wikiticker"},
+        dateTimeZone = "UTC"
     },
     air_quality = {
+        type = "abridged",
         metrics = {"averageCOPerDay", "averageNO2PerDay"},
         dimensions = air_quality_dimensions,
-        granularity = {"all", "hour", "day"},
+        granularities = {
```

Comment:
default retention to "P1Y"

Reply:
Build using GranularityDictionary, supplied by getGranularityDictionary from AbstractBinderFactory

Reply:
done.